modules/service/pom.xml

```diff
@@ -52,5 +58,24 @@
             <version>${aetherProviderVersion}</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.mule.tools.maven</groupId>
+            <artifactId>mule-packager</artifactId>
+            <exclusions>
+                <!--
+                Excluding dependencies that are required to generate the classloader-model.json that are not required on runtime.
```

Comment:
The functionality that requires these dependencies should be on a separate module so they don't have to be excluded by its clients. Create a JIRA to track this.